examples/motoko_examples/superheroes/.eslintrc

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
If you could remove this file

Reply:
This was added to fix eslint errors, it doesn't affect the build but it will show on the IDE as errors.
I will remove it either way.